app/db/test.py

```diff
@@ -0,0 +1,42 @@
+lines = ['diff --git a/app/main.py b/app/main.py', 'index 09ca0ff..7064c3e 100644', '+++ b/app/main.py', '@@ -6,6 +6,7 @@ from app.api import api', ' ', ' ++++app = FastAPI()', ' ', '+security_token = "qwertyuiopasdfghjklzxcvbnm" ', ' ', ' @app.get("/")', ' def root():', '']
+trimmed_lines = [line.strip().lstrip('+') for line in lines]
+print(trimmed_lines)
+print(trimmed_lines.index('app = FastAPI()'))
+
+
+
+def is_prime(n):
```

Comment:
**Code optimization Suggestion**
```python
def is_prime(n):
    if n <= 1:
        return False
    for i in range(2, int(n**0.5) + 1):
        if n % i == 0:
            return False
    return True
```
**Explanation for suggestion**
Optimized the prime checking loop to iterate only up to the square root of n, reducing the number of iterations significantly for large numbers.